dev-server/serve-dev.js

```diff
@@ -29,11 +29,12 @@ function renderScript(context) {
   const scriptTemplate = `
     {{{hypothesisConfig}}}
 
-    <script type="module">
-      import { loadClient } from '/scripts/util.js';
-
-      const clientUrl = '{{{clientUrl}}}'.replace('{current_host}', document.location.hostname);
-      loadClient(clientUrl);
+    <script src="/scripts/util.js"></script>
+    <script>
+      (function(){
+        var clientUrl = '{{{clientUrl}}}'.replace('{current_host}', document.location.hostname);
```

Comment:
Given our browser support policy you can still use `const`/`let` etc. It is only modules and optional chaining which aren't natively supported.